client/extensions/woocommerce/style.scss

```diff
@@ -6,19 +6,13 @@
 }
 
 .woocommerce {
+	@import 'app/shipping/style';
```

Comment:
Could not agree more that it should only be applied to body or the highest level container.